graphql_compiler/compiler/emit_match.py

```diff
@@ -198,3 +198,51 @@ def emit_code_from_ir(match_query):
     query_data.appendleft(_construct_output_to_match(match_query.output_block))
 
     return u' '.join(query_data)
+
+
+def emit_code_from_ir(compound_match_query):
+    """Return a MATCH query string from a list of MATCH queries (compound match query)."""
```

Comment:
`a list of MATCH queries (compound match query)` is confusing. Just reference the `CompoundMatchQuery` namedtuple, and people can look up what that is when they are curious.